### PR TITLE
KAW-7678 Set up style guide

### PR DESCRIPTION
**Pull request by TomaszDziezykNetcentric**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #KAW-7678

Test URLs:
- Before: https://main--edge-delivery-bimota--netcentric.hlx.page/drafts/tdziezyk/
- After: https://kaw-7678-set-up-style-guide--edge-delivery-bimota--netcentric.hlx.page/drafts/tdziezyk/

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -77,26 +49,66 @@ header {
...
+  font-size: 1.25rem;
+  line-height: 1.4;
+  font-weight: 600;
+}
+
 h5,
-h6 {
-  font-family: var(--heading-font-family);
+.h5 {
+  font-size: 16px;
```
Comment (lakshmishriaswathnarayana) [inline styles/styles.css]: Can you make this rem as rest of them are in rem?

File: styles/styles.css [hunk 2]
```diff
@@ -77,26 +49,66 @@ header {
...
-h5 { font-size: var(--heading-font-size-s); }
-h6 { font-size: var(--heading-font-size-xs); }
+h6,
+.h6 {
+  font-size: 0.875rem;
+  line-height: 1.4;
+  font-weight: 600;
+}
+
+@media (width >= 960px) {
```
Comment (lakshmishriaswathnarayana) [inline styles/styles.css]: Move these styles for 960px to line 243, so all styles for 960 and more stays in one place

File: styles/styles.css [hunk 3]
```diff
@@ -12,61 +12,33 @@
 
 :root {
   /* colors */
-  --link-color: #035fe6;
-  --link-hover-color: #136ff6;
+  --link-color: #f00;
+  --link-hover-color: #1d1d1b;
   --background-color: white;
```
Comment (lakshmishriaswathnarayana) [inline styles/styles.css]: Can you change this to  `--c-primary-white: #fff`.  Use it in line 179 (for button text color)

File: styles/styles.css [hunk 4]
```diff
@@ -145,45 +155,51 @@ a:hover {
...
+  text-transform: uppercase;
+  border-radius: 0;
+  background-color: var(--button-color);
+  color: #fff;
+  border: 1px solid var(--button-color);
+}
+
+a.button:active {
+  background-color: #c00;
+  color: #fff;
```
Comment (lakshmishriaswathnarayana) [inline styles/styles.css]: This is not required as it's same as line 179

File: styles/fonts.css [hunk 5]
```diff
@@ -1,18 +1,9 @@
...
-
-@font-face {
-  font-family: roboto;
+  font-family: Oxygen;
   font-style: normal;
   font-weight: 400;
   font-display: swap;
-  src: url('../fonts/roboto-regular.woff2') format('woff2');
-  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+  src: url("https://fonts.gstatic.com/s/oxygen/v15/2sDfZG1Wl4LcnbuKjk0mRUe0Aw.woff2") format('woff2');
```
Comment (lakshmishriaswathnarayana) [inline styles/fonts.css]: Can we add the fonts in local under folder fonts and use that url here ?

**Review by lakshmishriaswathnarayana — CHANGES_REQUESTED** [review]

File: styles/styles.css (hunk 1 above)
Comment (TomaszDziezykNetcentric) [reply styles/styles.css]: Done

File: styles/styles.css (hunk 2 above)
Comment (TomaszDziezykNetcentric) [reply styles/styles.css]: Moved

File: styles/styles.css (hunk 3 above)
Comment (TomaszDziezykNetcentric) [reply styles/styles.css]: Chagned

File: styles/styles.css (hunk 4 above)
Comment (TomaszDziezykNetcentric) [reply styles/styles.css]: Removed

File: styles/fonts.css (hunk 5 above)
Comment (TomaszDziezykNetcentric) [reply styles/fonts.css]: Local file added

**Review by lakshmishriaswathnarayana — APPROVED** [review]

**Pull request merged**